core/templates/dev/head/pages/creator_dashboard/CreatorDashboard.js

```diff
@@ -47,14 +47,15 @@ oppia.constant('HUMAN_READABLE_SUBSCRIPTION_SORT_BY_KEYS', {
 });
 
 oppia.controller('CreatorDashboard', [
-  '$scope', '$rootScope', '$window', 'oppiaDatetimeFormatter', 'alertsService',
-  'CreatorDashboardBackendApiService', 'RatingComputationService',
-  'ExplorationCreationService', 'UrlInterpolationService', 'FATAL_ERROR_CODES',
+  '$scope', '$rootScope', '$http', '$window', 'oppiaDatetimeFormatter',
+  'alertsService', 'CreatorDashboardBackendApiService',
+  'RatingComputationService', 'ExplorationCreationService',
+  'UrlInterpolationService', 'FATAL_ERROR_CODES',
   'EXPLORATION_DROPDOWN_STATS', 'EXPLORATIONS_SORT_BY_KEYS',
   'HUMAN_READABLE_EXPLORATIONS_SORT_BY_KEYS', 'SUBSCRIPTION_SORT_BY_KEYS',
   'HUMAN_READABLE_SUBSCRIPTION_SORT_BY_KEYS',
   function(
-      $scope, $rootScope, $window, oppiaDatetimeFormatter, alertsService,
+      $scope, $rootScope, $http, $window, oppiaDatetimeFormatter, alertsService,
```

Comment:
Please align line breaks here with those in the stringified list above -- easier to compare.

Reply:
Done